packages/orderbook/scripts/buildPackage.js

```
@@ -47,12 +48,12 @@
 const bytes = require('fs').readFileSync(path);`,
   `
 const { Buffer } = require('buffer');
-const wasmB64 = require('../${package}.json');
+const wasmB64 = require('../orderbook_wbg.json');
 const bytes = Buffer.from(wasmB64.wasm, 'base64');`
 );
 cjs = cjs.replace("const { TextEncoder, TextDecoder } = require(`util`);", "");
 cjs = "/* this file is auto-generated, do not modify */\n" + cjs;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)